### PR TITLE
FOGL-4174: fix a problem of deleted files during the upgrade

### PR DESCRIPTION
**Pull request by sstefano69**
Signed-off-by: Stefano <stefano@dianomic.com>

### CODE REVIEW

File: packages/Debian/common/DEBIAN/postrm [hunk 1]
```diff
@@ -35,8 +35,8 @@ remove_fledge_sudoer_file() {
     rm -rf /etc/sudoers.d/fledge
 }
 
-echo "Cleanup of files"
-remove_unused_files
+#echo "Cleanup of files"
+#remove_unused_files
```
Comment (ashish-jabble) [inline packages/Debian/common/DEBIAN/postrm:39]: Revert the changes as upgrade handled changes made into preinst. 
And this function improvisation will be via FOGL-3424

**Comment by ashish-jabble** [comment]
@MarkRiddoch fyi, detailed root cause findings available on FOGL-3424. Its debian flow which I didn't know before.

File: packages/Debian/common/DEBIAN/preinst [hunk 2]
```diff
@@ -76,6 +76,9 @@ if [ "$IS_FLEDGE_INSTALLED" -eq "1" ]
 then
     echo "Fledge is already installed: this is an upgrade/downgrade."
 
+    # Avoid the execution of the function during an upgrade
+    sed -i -e 's/^remove_unused_files$/#remove_unused_files/' /var/lib/dpkg/info/fledge.postrm
```
[the remark this answers is not in the document]
Comment (sstefano69) [reply packages/Debian/common/DEBIAN/preinst]: done

File: packages/Debian/common/DEBIAN/postrm (hunk 1 above)
Comment (sstefano69) [reply packages/Debian/common/DEBIAN/postrm:39]: I think we need this, as an upgrade is never  possible having remove_unused_files.

than eventually a following task will apply a different solution.

Comment (ashish-jabble) [reply packages/Debian/common/DEBIAN/postrm:39]: I would suggest you better to completely remove only the function call from here instead of commented code lines.
And function definition stay as it is; We have the FOGL-3424 to handle this situation in same function.

Comment (sstefano69) [reply packages/Debian/common/DEBIAN/postrm]: done
